packages/teams-js/src/public/meeting.ts

```diff
@@ -567,6 +591,9 @@ export namespace meeting {
   }
 
   /**
+   * @throws error if your app manifest does not include the `OnlineMeeting.ReadBasic.Chat` RSC permission.
```

Comment:
>OnlineMeeting.ReadBasic.Chat [](http://example.com/codeflow?start=62&length=28)

I didn't see `OnlineMeeting.ReadBasic.Chat` mentioned at the URL provided. Is it actually `OnlineMeetingParticipant.Read.Chat` or is that something different?

Reply:
Yeah, it was supposed to be documented there, but it seems like it only contains a subset of RSCs. I will add another link to the complete list of RSCs.

Thanks for catching this!

Reply:
Were you going to add a different link here?

Reply:
Yup! Wanted to batch the changes in a commit.

I have just pushed the changes.